frontend/src/pages/TermPlanner/SettingsMenu/SettingsMenu.tsx

```diff
@@ -28,7 +30,16 @@ const SettingsMenu = () => {
   }
 
   function handleSummerToggle() {
-    dispatch(toggleSummer());
+    try {
+      axios.put('/user/toggleSummerTerm', {}, { params: { token } });
+    } catch {
+      openNotification({
+        type: 'error',
+        message: 'Error setting summer term',
+        description: 'An error occurred when toggling the summer term.'
+      });
+      return;
+    }
```

Comment:
Make async function/await the request - error message looks fine to me

Reply:
Fixed